packages/server/assets/acc/typedefs/accSyncItems.graphql

```diff
@@ -0,0 +1,101 @@
+extend type Project {
+  accSyncItems(cursor: String, limit: Int): AccSyncItemCollection!
+  accSyncItem(id: String!): AccSyncItem!
+}
+
+type AccSyncItemCollection {
+  items: [AccSyncItem!]!
+  totalCount: Int!
+  cursor: String
+}
+
+type AccSyncItem {
+  id: ID!
+  projectId: String!
+  modelId: String!
+  accRegion: String!
+  accHubId: String!
+  accProjectId: String!
+  accRootProjectFolderUrn: String!
+  accFileLineageUrn: String!
+  accFileName: String!
+  accFileExtension: String!
+  accFileVersionIndex: Int!
+  accFileVersionUrn: String!
+  accFileViewName: String
+  accWebhookId: String
+  status: AccSyncItemStatus!
+  author: LimitedUser
+  createdAt: DateTime!
+  updatedAt: DateTime!
+}
+
+enum AccSyncItemStatus {
+  PENDING
```

Comment:
I'd want to avoid the situation where we have to map this value internally to a differently cased version, like how SortDirection ASC/DESC has to be mapped to asc/desc for Knex. 

If the default enum values aren't enough, you can adjust the enum typedef to have values different from the keys and you can also configure codegen to map to different values. Ideally, however, the enum keys should have the exact same casing/value as the values.